.github/scripts/previous-releases.jq

```diff
@@ -0,0 +1,23 @@
+# Extract only the `tag_name` field from each release object
```

Comment:
Optional:
I'd add an example of data (maybe even fake) that we expect for input